@Test public void should_fail_if_nothing_is_thrown_by_lambda(){
  assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> assertThatThrownBy(() -> {
  }
)).withMessage(format("%nExpecting code to raise a throwable."));
}
